### PR TITLE
Add regression test for XGboost with Sci-kit learn

### LINKED ISSUE #20: Improve OpenML docker image for scikit xgboost
The OpenML docker image in https://hub.docker.com/r/feedzai/oracle-jep-miniconda/ must be updated to also install xgboost dependencies for it to work in scikit.

This tutorial can be used to solve the problem: https://machinelearningmastery.com/develop-first-xgboost-model-python-scikit-learn/

After that, we should add a regression test for https://github.com/feedzai/feedzai-openml-python/issues/18 as suggested in https://github.com/feedzai/feedzai-openml-python/pull/19 that uses the model of https://files.feedzai.com/index.php/s/rKbif8CFpee79Gd

### PR DESCRIPTION
**Pull request by nmldiegues**
closes #20

### CODE REVIEW

File: openml-scikit/src/test/java/com/feedzai/openml/scikit/ScikitXGboostTest.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            TEST_SCHEMA,
+            TEST_DATASET_INSTANCES_SIZE,
+            new Random(123456789)
+    );
+
+    /**
+     * Simple regression test for https://github.com/feedzai/feedzai-openml-python/issues/20
+     */
+    @Test
+    public void test() throws ModelLoadingException {
```
[the remark this answers is not in the document]
Comment (nmldiegues) [reply openml-scikit/src/test/java/com/feedzai/openml/scikit/ScikitXGboostTest.java]: @pedrorijo91  the test is using assertJ assertThat; what should we do about this?

**Comment by nmldiegues** [comment]
I had forgotten to make a release of the Docker image https://hub.docker.com/r/feedzai/oracle-jep-miniconda/builds/ and to make the travis script point to the new version. Re-trying

**Comment by nmldiegues** [comment]
The problem seems to be that the re-built docker image now contains Python 3.7 rather than 3.6. I will have to fix the python version there because the conda environment is being created with just "python3" rather than "python3.6".

**Comment by pedrorijo91** [comment]
just a minor detail @nmldiegues: do you think it's worthy to "also add to the documentation that some models may require extra steps besides those we describe" ? just to keep users aware that just like xgboost requires extra installation steps, others may also need (should we have a test for each algorithm?)

**Comment by nmldiegues** [comment]
@pedrorijo91  that's already in the README:

image

**Comment by nmldiegues** [comment]
I heard 3rd works like a charm! Fingers crossed :)

**Review by pedrorijo91 — APPROVED** [review]

**Comment by nmldiegues** [comment]
Cherry-picked to hf-0.1.X

**Pull request merged**